### PR TITLE
Unsafe reflection java

### PR DESCRIPTION
**Pull request by xcorail**
This query has been submitted via the bounty program to our standard queries, but will raise too many false positives. Therefore we put it into the community queries.

### CODE REVIEW

**Comment by xcorail** [comment]
Hey reviewers @yo-h @nicowaisman 👋 Gentle ping

File: CodeQL_Queries/java/UnsafeReflection/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Overview
```
Comment (nicowaisman) [inline CodeQL_Queries/java/UnsafeReflection/README.md]: Can we add the author of the Query?

**Comment by yo-h** [comment]
Now that we have the `experimental` directory in the Semmle/ql repository, we could potentially merge it there rather than here (but changing local input to remote input and fixing the metadata).

**Comment by xcorail** [comment]
@yo-h I thought everything in `experimental` was supposed to be analyzed and eventually be either improved and merged, or rejected. If not, then yes, I'd put in into experimental!  
@nicowaisman ok I will add the author of the query in the readme

**Comment by yo-h** [comment]
> I thought everything in experimental was supposed to be analyzed and eventually be either improved and merged, or rejected.

I think it's fine to keep certain queries in `experimental`. For example, queries that intrinsically have a high FP rate that is difficult to improve sufficiently to move into the standard query set, but that may still have value for security researchers.

Let's hold off on this for now though, there are currently some follow-up discussions on the `experimental` workflow. Once they have come to a conclusion, we can decide whether to merge this PR, or move to `experimental` instead.

**Comment by aschackmull** [comment]
I think this is the sort of query where we'd like to see at least one plausible-looking result before merging it anywhere.

**Pull request closed without merging**